scripts/deploy.sh

```diff
@@ -134,11 +136,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
       echo "apply_security_hardening: false" >> $OA_OVERRIDES
     fi
 
+    # set the affinity to 3 for infra cluster (necessary for maas testing)
+    awk '{print} $0 == "  aio1:" && previous == "shared-infra_hosts:" \
+         {print "    affinity:\n      galera_container: 3\n      rabbit_mq_container: 3"} \
+         {previous = $0}' \
+    /etc/openstack_deploy/openstack_user_config.yml > /tmp/tmp && mv /tmp/tmp /etc/openstack_deploy/openstack_user_config.yml
```

Comment:
Thanks! I made a change to use this override in commit https://github.com/rcbops/rpc-openstack/pull/1577/commits/f84122a4873226f12673194f6416d0f9a803c29f